azurerm/internal/services/apimanagement/api_management_resource.go

```diff
@@ -284,7 +293,63 @@ func resourceApiManagementService() *schema.Resource {
 							Default:  false,
 						},
 
+						// TODO: Remove after deprecation
```

Comment:
We usually write the next major version in the TODO comment to aid our future selves :)

```suggestion
						// TODO: Remove in v3.0
```